charts/plum-batch/values.yaml

```diff
@@ -1,3 +1,4 @@
-image: hmctssandbox.azurecr.io/hmcts/plum-batch:latest
-replicas: 1
-schedule: "*/1 * * * *"
+job:
+  kind: CronJob
+  image: hmctssandbox.azurecr.io/hmcts/plum-batch:latest
```

Comment:
Is this a sensible default? I just put this to make the spike work - maybe `hmcts.azurecr.io` would be better?